android/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java

```diff
@@ -165,6 +171,25 @@ private static Uri parseAndVerifyUrl(String url) {
      */
     public WebResourceResponse shouldInterceptRequest(WebResourceRequest request) {
         Uri loadingUrl = request.getUrl();
+
+        if (null != loadingUrl.getPath()) {
```

Comment:
this can be a single if with both conditions since `return handleCapacitorHttpRequest(request, false);` should be changed to `return handleCapacitorHttpRequest(request);` as the boolean is not being used.